pkg/azurefile/azurefile.go

```diff
@@ -452,7 +452,12 @@ func (d *Driver) Run(ctx context.Context) error {
 	csi.RegisterControllerServer(server, d)
 	csi.RegisterNodeServer(server, d)
 	d.server = server
+	val, val2, err := getNodeInfoFromLabels(ctx, d.NodeID, d.kubeClient)
```

Comment:
you could define a member var of Driver struct, e.g. isKataNode and only run `isKataNode` func once during node initialization only if nodeID is not empty